### PR TITLE
Add parsing links logic 

### PR DESCRIPTION
**Pull request by chrisdns**

### CODE REVIEW

File: src/util/url-utils.ts [hunk 1]
```diff
@@ -40,3 +42,27 @@ export const parse = (header: string): any => {
...
   return links;
 };
+
+/**
+ * Retrieve new data when infinite scrolling
+ * @param currentData
+ * @param incomingData
+ * @param links
+ */
+export const loadMoreData = (currentData: any, incomingData: any, links: any): any => {
```
Comment (deepu105) [inline src/util/url-utils.ts]: can we rename it to `loadMoreDataWhenScrolled`

Comment (deepu105) [reply src/util/url-utils.ts]: Also, can we rename the `parse` method you added here to `parseHeaderForLinks`

Comment (chrisdns) [reply src/util/url-utils.ts]: maybe we should move loadMoreDataWhenScrolled move to another more appropriate file ?

**Comment by deepu105** [comment]
@chrisdns its released

**Pull request merged**